cmd/opentelemetry-collector/app/exporter/kafka/exporter.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafka
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/exporter"
+	"github.com/uber/jaeger-lib/metrics"
+	"go.uber.org/zap"
+
+	storageOtelExporter "github.com/jaegertracing/jaeger/cmd/opentelemetry-collector/app/exporter"
+	"github.com/jaegertracing/jaeger/plugin/storage/kafka"
+	"github.com/jaegertracing/jaeger/storage"
+)
+
+// New creates new Kafka exporter
+func New(config *Config, log *zap.Logger) (exporter.TraceExporter, error) {
+	f := kafka.NewFactory()
+	f.InitFromOptions(config.Options)
+	err := f.Initialize(metrics.NullFactory, log)
+	if err != nil {
+		return nil, err
+	}
+	return create(f, config)
```

Comment:
Only this line is not being covered from the whole PR. I cannot pass the initialization, there is no way to change the objects in the Kafka config for mocked builders.

Reply:
@yurishkuro any idea how to fix the coverage? Even though only a single line is not covered the total is: 

```
ok  	github.com/jaegertracing/jaeger/cmd/opentelemetry-collector/app/exporter/kafka	0.767s	coverage: 94.7% of statements

```